src/cljs/org/broadinstitute/firecloud_ui/page/workspace/details.cljs

```diff
@@ -53,6 +53,33 @@
                         (swap! state assoc :status :success :protected? protected?))
                      (swap! state assoc :status :error :message status-text)))}))})
 
+(react/defc BucketBanner
+  {:render
+   (fn [{:keys [props]}]
+    [:div {:style {:position "relative"}}
+     (case (:bucket-access? props)
+       nil [:div {:style {:position "absolute" :marginTop "-1.5em"}}
+            [comps/Spinner {:height "1.5ex"}]]
+       true nil
+       false [:div {:style {}}
+             [:div {:style {:height 1 :backgroundColor "#bbb" :marginBottom 2}}]
+             [:div {:style {:outlineTop (str "4px double #ccc")
+                            :backgroundColor "#ff8060"
+                            :fontSize "small"
+                            :padding "4px 0"
+                            :textAlign "center"}}
+              (if (= 404 (:status-code props))
```

Comment:
the 404 case wasn't a part of the story but it was easy enough to address. i think it's better behavior to let users know that their bucket doesn't even exist instead of displaying that as delayed access